packages/video_player/lib/video_player.dart

```diff
@@ -397,6 +405,21 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     value = value.copyWith(volume: volume.clamp(0.0, 1.0));
     await _applyVolume();
   }
+
+  static String _nameOf(VideoFormat format) {
+    switch (format) {
+      case VideoFormat.ss:
+        return 'ss';
```

Comment:
nits: Make these string constant, same on the android side.
e.g. kFormatSS, kFormatHLS...